dockstore-webservice/src/main/java/io/dockstore/webservice/core/metrics/MetricsByStatus.java

```diff
@@ -69,6 +69,12 @@ public class MetricsByStatus {
     @Schema(description = "Aggregated cost metrics in USD for the status")
     private CostStatisticMetric cost;
 
+    @Valid
+    @OneToOne(cascade = CascadeType.ALL, orphanRemoval = true)
```

Comment:
Should consider `@Embedded` instead if cascading all operations anyway. 
Along the lines of https://stackoverflow.com/a/49548539 

i.e. I don't think we'd ever consider reusing the time series for another workflow version for example, the times series would only ever be useful for this workflow version

Reply:
Agree that `@Embedded` might make sense, but recommend that for now, we stick with the separate table and cascading, because (by design) that it's similar to the way that the other `Metric` properties in this class work.  Would be curious to get Kathy's reasons as to the pros and cons of the two schemes.

Reply:
As to `@Embedded` versus `@OneToOne`,  if we made `TimeSeriesMetric` embeddable, I think we'd have to change all the Metric classes to be embeddable, because all the metrics extend `Metric`.  And maybe some other stuff:

https://in.relation.to/2024/07/12/embeddable-inheritance/

I would like to avoid combining a big refactor with the time series work, so recommend we stick with `@OneToOne` for now.  We can always do the refactor later.